back-end/src/main/java/index/Indexer.java

```diff
@@ -190,4 +204,28 @@ protected static String getPolygonText(double minx, double miny, double maxx, do
 
         return polygonText;
     }
+
+    protected static String getCubeText(double minx, double miny, double maxx, double maxy, int canvasId) {
```

Comment:
this function is something specific to cube indexer, thus supposed to be put in PsqlCubeIndexer. Functions in `Indexer.java` are common ones that are accessed by multiple Indexers.

Reply:
makes sense, I moved the function